Sources/CombineWamp/Model/URI.swift

```diff
@@ -60,8 +60,10 @@ public struct URI: Equatable, RawRepresentable, CustomStringConvertible, Lossles
 }
 
 extension URI {
-    enum WebSocketSubprotocol {
-        static var json = URI(unverified: "wamp.2.json")
-        static var msgpack = URI(unverified: "wamp.2.msgpack")
+    public struct SerializationFormat {
+        public let uri: URI
+
+        static var json = SerializationFormat(uri: URI(unverified: "wamp.2.json"))
```

Comment:
Don't know if that's intended or if I am having a brain hiccup right now, but to me this looks like these static vars are mutable, when they probably should not be.

Reply:
That's true, my fault here!